app/konnect/analytics/index.md

```diff
@@ -26,17 +26,17 @@ routes, the graphs show requests broken down by status codes.
 
 > _**Figure 1:** Graph showing throughput for a service with interval filter options._
 
-For greater insights into your service usage, access {% konnect_icon vitals %} [Vitals](https://cloud.konghq.com/vitals) <span class="badge plus"></span>.
+For greater insights into your service usage, access {% konnect_icon analytics %} [Analytics](https://cloud.konghq.com/vitals) <span class="badge plus"></span>.
```

Comment:
```suggestion
For greater insights into your service usage, access the dedicated {% konnect_icon analytics %} [Analytics](https://cloud.konghq.com/analytics) page <span class="badge plus"></span>.
```
If the URL is changing.

Also some rephrasing, since we seem to be referring to all of the graphs as part of "Analytics".